csrankings-s.csv

```diff
@@ -772,6 +772,7 @@ Shaoping Ma,Tsinghua University,http://www.tsinghua.edu.cn/publish/csen/4623/201
 Shaoqun Wu,University of Waikato,http://www.cms.waikato.ac.nz/people/shaoqun,RlOtjQoAAAAJ
 Shaoting Zhang,UNC - Charlotte,https://webpages.uncc.edu/~szhang16,oiBMWK4AAAAJ
 Shaowei Cai,Chinese Academy of Sciences,http://teacher.ucas.ac.cn/~caisw,mcuXqQoAAAAJ
+Shaowei Wang,University of Manitoba,https://sites.google.com/view/mambalab,PIBtljkAAAAJ&hl

```

Comment:
Name needs a disambiguation suffix (see https://dblp.org/pid/49/6937.html, probably `0002`).

Reply:
I have added the disambiguation suffix 0002 to Shaowei Wang